Session01/Session01/Program.cs

```diff
@@ -6,31 +6,50 @@
 
 namespace Session01
 {
-    public struct Animal
+    class Animal
     {
-        public String nume;
-        public String sunet;
+        public String name;
+        public String sound;
 
+        public void getSounds()
```

Comment:
1. The method name should be Pascal Case: GetSounds()
2. This method breaks some OOP principles: 
  2.1 encapsulation, since any consumer of the Animal API can change the name and sound of each animal
  2.2 Open Closed Principle